lib/improver/wind_gust_diagnostic.py

```diff
@@ -148,7 +148,7 @@ def extract_percentile_data(cube, req_percentile, standard_name):
         if not isinstance(cube, iris.cube.Cube):
             msg = ('Expecting {0:s} data to be an instance of '
                    'iris.cube.Cube but is'
-                   ' {1:s}.'.format(standard_name, type(cube)))
+                   ' {1:s}.'.format(standard_name, str(type(cube))))
```

Comment:
Another string formatting thing

Reply:
I've removed the `:s` format specifier and removed enforcing `type(cube)` to be a string.